README.md

```diff
@@ -69,6 +78,34 @@ const cast = await client.v2.publishCast(signerUuid, 'This is a test cast.');
 await client.v2.reactToCast(signerUuid, NeynarV2.ReactionType.Like, cast.hash);
 ```
 <!-- AUTO-GENERATED-CONTENT:END -->
+***Creating a Neynar Signer:***
+<!-- AUTO-GENERATED-CONTENT:START (CODE:src=./examples/neynarSigner.ts) -->
```

Comment:
It's odd that this example exists twice -- once here in the `farcaster-js` project root and then again in `farcaster-js-neynar`. Should we delete the duplicate and just have this README reference the one from the neynar package? Should we include this example in the top-level README at all, versus pointing users down to the neynar README if they need to create a neynar signer?

Reply:
I think pointing to the neynar readme makes sense - there's info on the CLI for quickly getting a signer in the top level readme. Anyone wanting to create neynar signers programmatically will be checking the docs for neynar anyways.